app/assets/stylesheets/shared/header.scss

```diff
@@ -257,12 +265,17 @@ header.navigation {
 
   .apply-button {
     @extend .button;
-    border-radius: 3px;
+    border-radius: 4px;
     display: inline;
     float: right;
-    font-weight: normal;
-    margin: 8px;
-    padding: 4px 16px;
+    font-weight: bold;
+    font-size: 14px;
+    margin: 8px 8px 10px 8px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `8px 8px 10px` instead of `8px 8px 10px 8px`